src/api.yaml

```diff
@@ -0,0 +1,127 @@
+  limit u32 undefined: Undefined U32 limit value
+  limit u64 undefined: Undefined U64 limit value
+  mip level count undefined: Undefined MIP level count
+  stride undefined: Undefined stride
+  whole map size: Value representing size of entire map
+  whole size: Value representing the whole size value
+
+enum:
+  adapter type:
+    description: Provides information on the type of adapter.
+    values:
+      discrete GPU: A separate discrete GPU adapter
+      integrated GPU: An integrated GPU / CPU adapter
+      CPU: A CPU adapter
+      unknown: A unknown adapter type
+
+  address mode:
+    description: >
+      Describes the behaviour of the sampler if the sampler footprint extends
+      beyond the bounds of the sampled texture.
+    ref:
+      name: GPUAddressMode
+      anchor: '#enumdef-gpuaddressmode'
+    values:
+      repeat: Texture coordinates wrap to the other side of the texture
+      mirror repeat: >
+          Texture coordinates wrap to the other side of the texture, but the
+          texture is flipped when the integer part of the coordinate is odd.
+      clamp to edge: Texture coordinates are clamped between 0.0 and 1.0, inclusive.
+
+  alpha mode:
+    description: >
+      Selects how the contents of the surface will be interpreted when read,
+      when rendered to the screen or used as an image source.
+    ref:
+      name: GPUCanvasAlphaMode
+      anchor: '#GPUCanvasAlphaMode'
+    values:
+      premultiplied: >
+          Read RBGA as premultipled: color values are premultipled by their
+          alpha value.
+      unpremultiplied: >
+          Read RGBA as not premulitipled: Color values are not premultipled by
+          their alpha value
+      opaque: Read RGB as opaque and ignore alpha value
+
+  backend type:
+    description: >
+      Specifies the type of backend to use with Dawn.
+    ref:
+    values:
+      'null': A null backend
+      WebGPU: The WebGPU backend
+      D3D11: The D3D11 backend
+      D3D12: The D3D12 backend
+      metal: The Metal backend
+      vulkan: The Vulkan backend
+      openGL: The OpenGL backend
+      openGLES: The OpenGL ES backend
+
+bitmask:
+  texture usage:
+    description: Determine how a GPUTexture maybe be used after creation.
+    ref:
+      name: GPUTextureUsage
+      anchor: '#namespacedef-gputextureusage'
+    values:
+      none: No usage specified.
+      copy src: The texture can be used as the source of a copy operation.
+      copy dst: The texture can be used as the destination of a copy operation.
+      texture binding: >
+        The texture can be bound for use as a sampled texture in a shader.
+      storage binding: >
+        The texture can be bound for use as a storage texture in a shader.
+      render attachment: >
+        The texture can be used as a colour or depth/stencil attachment in a
+        render pass.
+      present: The texture can be used to "present" the texture on screen.
```

Comment:
That usage doesn't really exist and should be removed from Dawn.

Reply:
File https://bugs.chromium.org/p/dawn/issues/detail?id=1535 for the removal.

Reply:
This will get removed after Dawn gets updated.